CONTRIBUTING.md

```diff
@@ -134,25 +133,39 @@ further stub generation requires manual invocation of the following command.)
 bazel run //build:gen_proto
 ```
 
-## Running Unit Tests
+## Testing
+
+### Testing Conventions
+
+All python unit tests in this repo is based on Tensorflow's
+[tf.test.TestCase](https://www.tensorflow.org/api_docs/python/tf/test/TestCase),
+which is a subclass of
+[py-absl TestCase](https://github.com/abseil/abseil-py/blob/06edd9c20592cec39178b94240b5e86f32e19768/absl/testing/absltest.py#L523).
+
+We have several types of tests in this repo:
 
-At this point all unit tests are safe to run externaly. We are working on
+* Unit tests for source code;
+* End to end tests (filename ends with `_e2e_test.py`): some of this also runs with
+  external environments.
+
+### Running Unit Tests
+
+At this point all unit tests are safe to run externally. We are working on
 porting the end to end tests.
 
 Each test can just be invoked with `python`. To invoke all unit tests:
 
 ```shell
-find . -name '*_test.py' | grep -v e2e | xargs -I {} python {}
+find ./tfx -name '*_test.py' | grep -v e2e | xargs -I {} python {}
```

Comment:
note that many people set up a venv under `./.venv`, so this avoids the command finding tests that are packaged alongside tfx dependencies